.env.example

```
@@ -20,4 +20,5 @@ NEXT_PUBLIC_HASURA_GRAPHQL_URL=http://localhost:8080/v1/graphql
 # REMOVED FOR SECURITY: NEXT_PUBLIC_HASURA_ADMIN_SECRET=myadminsecretkey
 
 # Development
-NODE_ENV=development
\ No newline at end of file
+NODE_ENV=development
+NEXT_PUBLIC_BACKEND_URL=http://localhost:3001
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)